01-data-structures/02-stacks-and-queues/myqueue/myqueue.rb

```diff
@@ -8,11 +8,17 @@ def initialize
   end
 
   def enqueue(element)
+    enqueue = @queue.insert(0, element)
```

Comment:
The `enqueue = ` is redundant here. You never actually use the value of `enqueue`.